config/routes.rb

```diff
@@ -382,8 +382,8 @@
           # sometimes underscores get messed up when linked to via SMS
           as: :capture_doc_dashes
 
-      get '/in_person_proofing/ssn' => 'in_person/ssn#show'
-      put '/in_person_proofing/ssn' => 'in_person/ssn#update'
+      get '/in_person/ssn' => 'in_person/ssn#show'
+      put '/in_person/ssn' => 'in_person/ssn#update'
```

Comment:
In the PR where you change this url, suggest adding redirects under deprecated routes (near line 409):
      get '/in_person_proofing/ssn', to: redirect('/verify/in_person/ssn')
      put '/in_person_proofing/ssn', to: redirect('/verify/in_person/ssn')